setup.py

```diff
@@ -36,7 +36,7 @@
     ],
     keywords='cli',
     packages=find_packages(exclude=['docs', 'tests*']),
-    install_requires=['docopt'],
+    install_requires=['docopt', 'pylint'],
```

Comment:
**thought**: Try to keep developer requirements separate from script requirements